<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateStudentAccountWindow</class>
 <widget class="QDialog" name="CreateStudentAccountWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Student Account</string>
  </property>
  <widget class="QLabel" name="statusLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="firstNameLE">
      <property name="placeholderText">
       <string>Your First Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lastnameLE">
      <property name="placeholderText">
       <string>Your Last Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="idLE">
      <property name="placeholderText">
       <string>Your ID</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="submitBtn">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
